Maarkis/MeAgendaAi
# Company registration report

- Encrypt password
- Refactoring of validate method of entity
- Archive moved
- Deleted archive
- Report implementation [no-test]
- Testing repository company and moving folder
- Moved archives of security folder
- Message commit
- Test for GetAllAsync method.
- - Get All in services generic created
- Company report method created
- Created extension methods for LINQ
  - IsEmpty method
  - Test for IsEmpty method
- Adjusted test of Add Company
- Report service created
- - Method created 'Without Surname' at the Name Object Builder Extensions
- Adjusted CompanyBuilder for not set surname
- Tests created for Company Report
- Adjusted TestBase Class
   -  SetUp Async created in TestBase
- Adjusted tests in ReportCompanyService
- Fixed test ReportAsync_ShouldGenerateReportCorrectly
- Swapping Append for AppendLine
- Test skipped [ReportCompany_ShouldCorrectlyGenerateReport]
- Adjusted Header method
- Add Trim in CSV expected and result
Adjusted ReportCompany test
- New Line adjusted in test
- Solving possible null returns
- Adjusted filename in controller
Created test for filename
- Removing virtual from DbSet.
- Renaming method
- Field name adjusted from TIME_LIMITCANCELHOURS to TIME_LIMIT_CANCEL_HOURS
- Adjusted property name
Adjusted if null
- Adjusted CompanyController Test
